runtime.md

```diff
@@ -16,6 +16,7 @@ There is no requirement that it be unique across hosts.
 The ID is provided in the state because hooks will be executed with the state as the payload.
 This allows the hooks to perform cleanup and teardown logic after the runtime destroys its own state.
 * **`pid`**: (int) is the ID of the main process within the container, as seen by the host.
```

Comment:
> We could change all state fields phrased like: pid: (int, OPTIONAL)...

I have that in flight with #507.  I'll pull that apart to isolate it from the `creating` change.